### DISCUSSION TITLE
implement bufferToString?

### DISCUSSION THREAD

**Discussion by randrrandr1** [denoland/deno#16751] [Ideas]
Hello!
How about this performance improvement?
https://github.com/nodejs/performance/issues/16

https://github.com/nodejs/node/pull/45562/files#diff-81f77f34c4f5b8cc35f1a506808b94e2a281f3341c0f1a38ef2b8a8ba730b22bR523

**Comment by cjihrig** [comment]
It looks like that is specific to the Node.js `Buffer`, which does not exist in Deno. If it ends up in Node, we could add it to Deno's Node compatibility code, but that is implemented in JS, not C++, so it might not lead to much benefit other than compatibility.
